ouroboros-consensus-test/test-consensus/Test/Consensus/HardFork/Combinator/A.hs

```diff
@@ -199,8 +198,24 @@ data PartialLedgerConfigA = LCfgA {
     }
   deriving NoThunks via OnlyCheckWhnfNamed "LCfgA" PartialLedgerConfigA
 
-type instance LedgerCfg (LedgerState BlockA) =
-    (EpochInfo Identity, PartialLedgerConfigA)
+instance ToCBOR PartialLedgerConfigA where
+  toCBOR (LCfgA k systemStart forgeTxs) = mconcat [
+      encodeListLen 3
+    , toCBOR k
+    , toCBOR systemStart
+    , encode forgeTxs
+    ]
+
+instance FromCBOR PartialLedgerConfigA where
+  fromCBOR = do
+    enforceSize "PartialLedgerConfigA" 3
+    LCfgA <$> fromCBOR <*> fromCBOR <*> decode
+
+instance SerialiseNodeToClient BlockA PartialLedgerConfigA where
+  encodeNodeToClient _ _ = toCBOR
+  decodeNodeToClient _ _ = fromCBOR
+
+type instance LedgerCfg (LedgerState BlockA) = PartialLedgerConfigA
```

Comment:
This naming is incongruous. We should probably rename `PartialLedgerConfigA` to `LedgerConfigA`